### ISSUE TITLE
how to use other loss functions?

### ISSUE THREAD

**Issue by 812130247** [albumentations-team/autoalbument#50]
Can I use other loss functions officially provided by pytorch, how should I set it in search.yaml？
